extensions/indexes/lucene/src/main/java/org/exist/indexing/lucene/LuceneFieldConfig.java

```diff
@@ -218,6 +234,89 @@ private Field convertToField(String content) {
         return null;
     }
 
+    private Field convertToDocValue(String content) {
+        try {
+            final byte[] data;
+            final XMLGregorianCalendar calendar;
+            final int ms;
+            final int timezone;
+            switch(type) {
+                case Type.TIME:
+                    final TimeValue tv = new TimeValue(content);
+                    calendar = tv.calendar;
+                    data = new byte[7]; // allocate an appropriately sized
```

Comment:
This can be written into just 5 bytes.

Reply:
No, don't forget the milliseconds and timezone, which both need to be short!

Reply:
By my calculations:

1. Hour = 5 bits
2. Minute = 6 bits
3. Second = 6 bits
4. Milliseconds = 10 bits
5. Timezone (Hour = 4 bits, Minute = 6 bits, ± = 1 bit) = 11 bits

Total = 38 bits, i.e. 5 bytes.